tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -54,6 +55,7 @@ export class PodmanDesktopRunner {
     this._videoAndTraceName = undefined;
     this._autoUpdate = autoUpdate;
     this._autoCheckUpdate = autoCheckUpdate;
+    this._testStatus = true;
```

Comment:
maybe better naming? Like testFailed, testPassed ?

Reply:
changed named to `_testPassed` for the field.